src/main/java/edu/gatech/cs2340/trydent/sample/SpriteExample.java

```diff
@@ -0,0 +1,63 @@
+package edu.gatech.cs2340.trydent.sample;
+
+import javafx.scene.paint.Color;
+import edu.gatech.cs2340.trydent.Behavior;
+import edu.gatech.cs2340.trydent.GameObject;
+import edu.gatech.cs2340.trydent.Sprite;
+import edu.gatech.cs2340.trydent.Time;
+import edu.gatech.cs2340.trydent.TrydentEngine;
+import edu.gatech.cs2340.trydent.math.Position;
+
+/**
+ * Example to show some possible ways to use sprites.
+ */
+public class SpriteExample implements Runnable {
+
+    public static void main(String[] args) {
+        TrydentEngine.start();
+        TrydentEngine.setWindowTitle("TrydentEngine - Sprite Example");
+        TrydentEngine.setWindowSize(720, 480);
+        TrydentEngine.setBackgroundColor(Color.WHITE);
+        TrydentEngine.runLater(new SpriteExample());
+    }
+
+    @Override
+    public void run() {
+        String prefix = "edu/gatech/cs2340/trydent/sample/";
+        GameObject pikachu = new Sprite(prefix + "pikapika.gif");
+        pikachu.setLocalPosition(new Position(50, 50));
+
+        Sprite pacman = new Sprite(
+            new String[]{
+                prefix + "wakawaka1.png",
+                prefix + "wakawaka2.png",
+                prefix + "wakawaka3.png",
+                prefix + "wakawaka2.png"
+            },
+            0.05,
+            "pacman"
+        );
```

Comment:
This would be much prettier if the constructor took in a var-args list of strings, I think.
Eg, 
public Sprite(double duration, String name, String ... filepaths);

new Sprite(0.5, "pacman", prefix+"wakawaka1.png", prefix+"wakawaka2.png", prefix+"wakawaka3.png", prefix+"wakawaka2.png");

Without preventing the existing use-case of passing in a string-array.